src/OctokitGitHub.test.ts

```diff
@@ -142,6 +142,14 @@ describe(OctokitGitHub.name, () => {
       )
     )
   })
+
+  it('get list of all repositories', async () => {
+    const gitHubClient = client()
+
+    const repoList = await gitHubClient.getRepositories()
+
+    assertThat(repoList, hasItem('.github'))
+  })
```

Comment:
This test is actually misleading a little bit, that it only check if a specific item is in the list. This is because I found it difficult to assert that the variable with the actually number of repositories (the number can change, we can't hard code it, or if we were to create dummy repositories to test, the running time will be quit a bit long,..)

I need more opinions on how should I improve or implement the test for this change.

Reply:
I think this is just fine to give us enough confidence that it's working, without being brittle to the real data changing, as you say. 👍🏻

Reply:
Maybe we could also check that the number of results is `> 30`?